cmd/loadTests.go

```diff
@@ -95,12 +105,17 @@ type LogData struct {
 	AverageTimeToRunPipelineSucceeded float64           `json:"runPipelineSucceededTimeAvg"`
 	MaxTimeToRunPipelineSucceeded     float64           `json:"runPipelineSucceededTimeMax"`
 	AverageTimeToRunPipelineFailed    float64           `json:"runPipelineFailedTimeAvg"`
+	AverageTimeToDeploymentSucceeded  float64           `json:"DeploymentSucceededTimeAvg"`
+	MaxTimeToDeploymentSucceeded      float64           `json:"DeploymentSucceededTimeMax"`
+	AverageTimeToDeploymentFailed     float64           `json:"DeploymentFailedTimeAvg"`
```

Comment:
```suggestion
	AverageTimeToDeploymentSucceeded  float64           `json:"deploymentSucceededTimeAvg"`
	MaxTimeToDeploymentSucceeded      float64           `json:"deploymentSucceededTimeMax"`
	AverageTimeToDeploymentFailed     float64           `json:"deploymentFailedTimeAvg"`
```
Please follow the naming convention here to keep it consistent.

Reply:
Updated